.github/workflows/CI.yml

```diff
@@ -41,6 +41,12 @@ jobs:
     if: needs.setup.outputs.markdown != 'false'
     uses: ./.github/workflows/validate-local-links-in-md.yml
 
+  jest:
+    name: Jest
+    needs: [setup]
+    if: needs.setup.outputs.frontend != 'false'
+    uses: ./.github/workflows/jest.yml
```

Comment:
Good point, great to hear @VIKTORVAV99 

It would be nice to have Jest in CI when as we're finishing up so we don't accidentally break something at the finish line

Reply:
Fair, this is only being merged into our temporary `historical-view-v1` branch, so there won't be any overlap just yet - but let's just merge both and remember to update this when we launch :)

Reply:
Since the other PR was merged first I suggest you update the needs array to match cypress (`needs:[eslint, verify]`) and remove the if statement all together when merging to master later.